src/ExtensionsSample/packages.config

```diff
@@ -6,6 +6,7 @@
   <package id="Microsoft.AspNet.WebHooks.Receivers" version="1.2.0-rc1" targetFramework="net45" />
   <package id="Microsoft.AspNet.WebHooks.Receivers.GitHub" version="1.2.0-rc1" targetFramework="net45" />
   <package id="Microsoft.Azure.Mobile.Client" version="2.0.1" targetFramework="net45" />
+  <package id="Microsoft.Azure.DocumentDB" version="1.5.3" targetFramework="net451" />
```

Comment:
I was testing a private nuget package from docdb that required net451. This must have stuck around when I backed it out. I've fixed it.

---
In reply to: [56355454](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/47#discussion_r56355454) [](ancestors = 56355454)